politeiawww/backend.go

```diff
@@ -1501,7 +1501,7 @@ func (b *backend) ProcessSetProposalStatus(sps www.SetProposalStatus, user *data
 }
 
 // ProcessProposalDetails tries to fetch the full details of a proposal from politeiad.
-func (b *backend) ProcessProposalDetails(propDetails www.ProposalsDetails, isUserAdmin bool) (*www.ProposalDetailsReply, error) {
+func (b *backend) ProcessProposalDetails(propDetails www.ProposalsDetails, user *database.User, isUserAdmin bool) (*www.ProposalDetailsReply, error) {
```

Comment:
I don't think the `isUserAdmin` parameter is necessary anymore because we can just pull it from `user`.